zerver/lib/actions.py

```diff
@@ -1781,28 +1781,28 @@ def build_message_send_dict(
     # Render our message_dicts.
     assert message.rendered_content is None
 
-    rendered_content = render_incoming_message(
+    result = render_incoming_message(
```

Comment:
Let's call this variable `rendering_result`; this is a long enough function that being explicit in variable names is helpful.